openmls/src/framing/mls_content_in.rs

```diff
@@ -41,10 +44,29 @@ pub(crate) struct FramedContentIn {
     pub(super) epoch: GroupEpoch,
     pub(super) sender: Sender,
     pub(super) authenticated_data: VLBytes,
-
     pub(super) body: FramedContentBodyIn,
 }
 
+impl FramedContentIn {
+    /// Returns a [`FramedContent`] after successful validation.
+    pub(crate) fn into_validated(
```

Comment:
`into_validated` generally sounds a bit clunky. Maybe just `validate` in the same vein as `verify`?

Reply:
I'm following the convention here: https://rust-lang.github.io/api-guidelines/naming.html#ad-hoc-conversions-follow-as_-to_-into_-conventions-c-conv

Reply:
I'm aware of those conventions, but I still think it's clunky and is not consistent with the existing use of `verify`. What I'm getting at is there in this particular instance, the point to is not to highlight the ownership, but the fact that we validate the given struct. In that sense, I believe that `validate` gives the reader a better idea of what's going on.